docker-compose.yml

```
@@ -0,0 +1,101 @@
+version: "3.4"
+services:
+  clickhouse:
+    image: yandex/clickhouse-server:20-alpine
+    ports:
+      - 8123:8123
+      - 9000:9000
+  impala:
+    depends_on:
+      - postgres
+    environment:
+      PGPASSWORD: postgres
+    healthcheck:
+      interval: 30s
+      retries: 20
+      test:
+        - CMD-SHELL
+        - nc -z 127.0.0.1 21050 && nc -z 127.0.0.1 50070
+      timeout: 10s
+    hostname: localhost
+    image: ibisproject/impala:latest
+    ports:
+      - 9020:9020
+      - 50070:50070
+      - 50075:50075
+      - 8020:8020
+      - 8042:8042
+      - 9083:9083
+      - 21000:21000
+      - 21050:21050
+      - 25000:25000
+      - 25010:25010
+      - 25020:25020
+  postgres:
+    environment:
+      POSTGRES_PASSWORD: postgres
+    healthcheck:
+      interval: 10s
+      retries: 3
+      test:
+        - CMD
+        - pg_isready
+      timeout: 5s
+    image: postgres:9.5-alpine
+    ports:
+      - 5432:5432
+  kudu:
+    cap_add:
+      - SYS_TIME
+    depends_on:
+      - impala
+      - kudu-tserver
+    environment:
+      KUDU_MASTER: "true"
+    image: ibisproject/kudu:latest
+    ports:
+      - 7051:7051
+      - 8051:8051
+  kudu-tserver:
+    cap_add:
+      - SYS_TIME
+    depends_on:
+      - impala
+    environment:
+      KUDU_MASTER: "false"
+    image: ibisproject/kudu:latest
+    ports:
+      - 7050:7050
+      - 8050:8050
+  mysql:
+    environment:
+      MYSQL_ALLOW_EMPTY_PASSWORD: "true"
+      MYSQL_DATABASE: ibis_testing
+      MYSQL_PASSWORD: ibis
+      MYSQL_USER: ibis
+    healthcheck:
+      interval: 10s
+      retries: 3
+      test:
+        - CMD
+        - mysqladmin
+        - ping
+      timeout: 5s
+    image: mariadb:10.4.22
+    ports:
+      - 3306:3306
+  ibis-postgres:
+    environment:
+      POSTGRES_PASSWORD: ""
+      POSTGRES_DB: ibis_testing
```

```suggestion
  ibis-postgres:
    user: postgres
    environment:
      POSTGRES_PASSWORD: ""
      POSTGRES_DB: ibis_testing
```

fix for the `role "root" does not exist` error